src/cdomain/value/cdomains/intDomain.mli

```diff
@@ -411,17 +407,17 @@ module Interval32 :Y with (* type t = (IntOps.Int64Ops.t * IntOps.Int64Ops.t) op
 
 module BigInt:
   sig
-    include Printable.S with type t = Z.t (* TODO: why doesn't this have a more useful signature like IntOps.BigIntOps? *)
```

Comment:
This comment should maybe remain if we need to keep `BigInt` (see previous comment).